data/json/npcs/other/homeless/group camp/group camp missions.json

```diff
@@ -0,0 +1,228 @@
+[
+{
+    "id": "MISSION_KILL_BLOBS",
+    "type": "mission_definition",
+    "name": "Sticky Situation",
+    "description": "Kill some blobs in exchange for credit with some of the traders.",
+    "goal": "MGOAL_FIND_ITEM",
+	"item": "slime_scrap",
+	"count": 6,

```

Comment:
Please mention amount of slime scraps you need to deliver in the description.

Reply:
> Please mention amount of slime scraps you need to deliver in the description.

That has been fixed.